packages/graphiql-react/package.json

```
@@ -13,18 +13,22 @@
   "license": "MIT",
   "main": "dist/index.js",
   "module": "dist/index.mjs",
+  "types": "dist/types/index.d.ts",
```

```suggestion
  "types": "dist/index.d.ts",
```

I wanna types lives just in `dist`

---

I couldn't get this working with the vite plugin for whatever reason. i think keeping them at `/types` as before is fine for now